requirements.txt

```diff
@@ -18,7 +18,7 @@ sphinx
 django-sortedm2m
 
 # REST API
-djangorestframework==2.3.13
+djangorestframework==3.1.3
```

Comment:
I don't like pinning to specific versions unless I know there are problems, I would suggest setting this to ``djangorestframework>=3.1,<3.2`` then we always get the latest in the 3.1.x branch, which in most cases just means bugfixes